docs/old/de/editexample6.html

```diff
@@ -164,7 +164,7 @@ <H2>
     erlaubt die Eingabe dieser Information.
   <P>
   Dann wird in jedem Zeichen, daß Anhänge besitzen soll, zunächst die
-  Stelle angeklickt, an der der Anker erstellt werden soll und dann der
+  Stelle angeklickt, an der Anker erstellt werden soll und dann der

```

Comment:
My rudimentary acquaintance with Deutsch seems to agree with gTranslate that this is not a repeated word. Second 'der' is the article for "the Anchor". First 'der' is sorta like the word 'which' or 'what'. Thus "at *which* *the* anchor"

Reply:
@tshinnic, you are correct.  There have to be two "der".